src/iota/__init__.py

```diff
@@ -17,6 +17,17 @@
 In that way, it's kind of like toxic waste in a superhero story.
 """
 
+STANDARD_UNITS = {
+    # Valid IOTA unit prefixes. Example value '25.2 Mi'
```

Comment:
... and *that's* the first thing that appears in this PR.  Great; put my pedantry on display front and center :P